src/main/java/racing/domain/car/Car.java

```diff
@@ -0,0 +1,40 @@
+package racing.domain.car;
+
+public class Car {
+
+    private static final int STANDARD_NUMBER_MOVE_FORWARD = 4;
+
+    private final Name name;
+    private Position position;
+
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public Car(String name, int value) {
+        this.name = new Name(name);
+        this.position = new Position(value);
+    }
+
+    public boolean isSamePosition(final Position position) {
+        return this.position.equals(position);
+    }
+
+    public void drive(int randomNumber) {
+        if (isMovable(randomNumber)) {
```

Comment:
isMovable 이라는 메서드를 따로 분리할 필요가 없는 것 같아요. 👀

```suggestion
        if (randomNumber >= STANDARD_NUMBER_MOVE_FORWARD) {
```